net-libs/libbitcoinconsensus/libbitcoinconsensus-0.15.1.recipe

```diff
@@ -0,0 +1,98 @@
+SUMMARY="Original Bitcoin client"
+DESCRIPTION="BitCoin Core is the backbone of the network. It downloads and, \
+by default, stores the entire history of Bitcoin transactions (which is \
+currently more than 100 GBs); depending on the speed of your computer and \
+network connection, the synchronization process can take anywhere from a few \
+hours to a day or more."
+HOMEPAGE="https://bitcoincore.org/"
+COPYRIGHT="2009-2017, Bitcoin Core Developers"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/bitcoin/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="98b3a98a67054123bef6714e4035e18f9250a1b6d63829258ad057906e75d8b1"
+SOURCE_DIR="bitcoin-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libbitcoinconsensus = $portVersion
```

Comment:
I can't see anything wrong that could make the Travis build fail.
The only thing I just noticed is that we are missing a `$secondaryArchSuffix` here.
```diff
 PROVIDES="
-	libbitcoinconsensus = $portVersion
+	libbitcoinconsensus$secondaryArchSuffix = $portVersion
```
But I don't think this could be cause of the failure.